discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -89,30 +94,34 @@ def _fill_tfvars(self, running=True):
         logging.info("Filling tfvars")
         with open(tfvars_json_file) as _file:
             tfvars = json.load(_file)
-        logging.info(self.network_conf.machine_cidr)
+
+        machine_cidr = self._get_machine_cidr()
+        provisioning_cidr = self._get_provisioning_cidr()
+
+        logging.info(machine_cidr)
```

Comment:
Please write a more descriptive log.
`"Machine cidr is: "`

Reply:
OK